Traceback (most recent call last):
  File "/home/runner/work/self-heal-CI-CD/self-heal-CI-CD/src/bug.py", line 4, in <module>
    subprocess.run("echo 'Hello, World!'", shell=True)
NameError: name 'subprocess' is not defined
